docker-compose.yaml

```diff
@@ -188,6 +188,13 @@ services:
       - POSTGRES_USER=${DATABASE_USER}
       - TEMPORAL_HISTORY_RETENTION_IN_DAYS=${TEMPORAL_HISTORY_RETENTION_IN_DAYS}
       - WORKSPACE_ROOT=${WORKSPACE_ROOT}
+      - DATABASE_PASSWORD=${DATABASE_PASSWORD}
+      - DATABASE_URL=${DATABASE_URL}
+      - DATABASE_USER=${DATABASE_USER}
+      - CONFIGS_DATABASE_MINIMUM_FLYWAY_MIGRATION_VERSION=${CONFIGS_DATABASE_MINIMUM_FLYWAY_MIGRATION_VERSION}
+      - REMOTE_CONNECTOR_CATALOG_URL=${REMOTE_CONNECTOR_CATALOG_URL}
+      - DEPLOYMENT_MODE=${DEPLOYMENT_MODE}
+      - UPDATE_DEFINITIONS_CRON_ENABLED=${UPDATE_DEFINITIONS_CRON_ENABLED}
```

Comment:
all of these should be sorted.

How do we want OSS users to interact with this? If this is turned on for them, we should also update .env file for users that run docker compose up and the kustomize file (since we are still working on Helm).

Reply:
For now, I don't expect OSS users to use this - they _could_  enable it and get auto-updating connectors, but it's mainly meant for cloud right now since that was our main focus. We're currently also not publishing an OSS catalog. I think we want to do something slightly different for officially bringing auto-updates to OSS (see evan's doc [here](https://docs.google.com/document/d/1SxsauSI2_mYduhLtHldWfjEGXIWtIiubwQInE0N0fcE/edit#)). I just added this in OSS because eventually the two upgrade mechanisms between cloud/oss will converge.